the linux one sometimes crashes, and this is optional/not required, so use the
stub fuschia one
https://gitlab.alpinelinux.org/alpine/aports/-/issues/15660
--
diff --git a/net/dns/BUILD.gn b/net/dns/BUILD.gn
index f36bf68..805d9a6 100644
--- a/net/dns/BUILD.gn
+++ b/net/dns/BUILD.gn
@@ -130,8 +130,8 @@ source_set("dns") {
     ]
   } else if (is_linux) {
     sources += [
-      "dns_config_service_linux.cc",
-      "dns_config_service_linux.h",
+      "dns_config_service_fuchsia.cc",
+      "dns_config_service_fuchsia.h",
     ]
   } else if (is_posix) {
     sources += [
